README.md

```diff
@@ -42,73 +47,126 @@ import 'material-design-icons-iconfont/dist/material-design-icons-no-codepoints.
 Vue.use(Vuetify)
 
 export function createApp() {
-    const app = new Vue({
-        vuetify: new Vuetify({
-            icons: {
-                iconfont: 'md',
-            },
-        }),
-        render: (h) => h(App),
-    });
-
-    return { app };
+ const app = new Vue({
+  vuetify: new Vuetify({
+   icons: {
+    iconfont: 'md',
+   },
+  }),
+  render: (h) => h(App),
+ });
+
+ return {app};
 }
 
 createApp().app.$mount('#app');
 ```
 
-### Through css import:
+### Through CSS import:
+
 ```scss
 // App.scss
 // Second way to import font
 @import "material-design-icons-iconfont/dist/material-design-icons-no-codepoints.css";
 ```
 
-### Through css file: 
+### Through CSS file:
+
 ```scss
 // material-design-icons-iconfont/dist/material-design-icons-no-codepoints.css
 @charset "UTF-8";
 @font-face {
   // See font name here
-    font-family: 'Material Icons';
-    src: url("./fonts/MaterialIcons-Regular.eot");
-    src: url("./fonts/MaterialIcons-Regular.woff2") format("woff2"),
-    url("./fonts/MaterialIcons-Regular.woff") format("woff"),
-    url("./fonts/MaterialIcons-Regular.ttf") format("truetype");
-    ...
+  font-family: 'Material Icons';
+  src: url("./fonts/MaterialIcons-Regular.eot");
+  src: url("./fonts/MaterialIcons-Regular.woff2") format("woff2"),
+  url("./fonts/MaterialIcons-Regular.woff") format("woff"),
+  url("./fonts/MaterialIcons-Regular.ttf") format("truetype");
+  ...
 }
 
 .material-icons {
-    ...
+  ...
+}
+
+```
+
+## Google fonts urls
+
+Plugin additionally supports a Google font url transforming for
+enabling [minification](https://developers.google.com/fonts/docs/getting_started?hl=en#optimizing_your_font_requests):
```

Comment:
`Plugin additionally supports Google font url transformation to enable [minification]`